### ISSUE TITLE
sst start doesn't work

### ISSUE THREAD

**Issue by AndreaNocciolini** [serverless-stack/sst#1976]
Hello!
This morning I tried to start a new project following the SST guide, like I did a few days ago. But this time, I cannot even start to work on the project: after initializing the project with `npx create-sst@latest --template=minimal/javascript-starter name-example` and install the npm packages inside the directory with `npm i`, I try to run the command `npm start`, but i obtain this error: </br>
![immagine](https://user-images.githubusercontent.com/80393968/185791263-c9c43c5c-ed94-4687-80d3-81864b8376cf.png)

The other day everything works correctly, I don't understand why it's not working now. Do you have any idea why is this happening?

**Comment by fwang** [comment]
Hey @AndreaNocciolini, could you open up `node_modules/@serverless-stack/core/assets/bootstrap/cdk-app/bin/index.mjs` and before line 10, add `console.log(process.argv)`?

ie.
```diff
+ console.log(process.argv);
  const region = process.argv[2];
  const tags = JSON.parse(process.argv[3]);
```

**Comment by fwang** [comment]
Hey @AndreaNocciolini, figured it out. It's an issue with Windows specifically.

Fixed in v1.8.3

Let me know if it works for you.

**Comment by AndreaNocciolini** [comment]
Hi @fwang,
it works perfectly now! Thank you very much!
